appveyor.yml

```diff
@@ -1,15 +1,20 @@
 environment:
   matrix:
+    - nodejs_version: "8.11.3"
     - nodejs_version: "8"
     - nodejs_version: "10"
+cache:
+ - "%LOCALAPPDATA%\\Yarn"
 install:
-  # get the latest version of Node.js in a release series
   - ps: Install-Product node $env:nodejs_version
-  # output useful info for debugging
+  - ps: Start-FileDownload 'https://github.com/yarnpkg/yarn/releases/download/v1.12.3/yarn-1.12.3.msi'
+  - cmd /c start /wait msiexec.exe /i yarn-1.12.3.msi /quiet /qn /norestart
+  - rm yarn-1.12.3.msi
   - node --version
-  - npm --version
-  # install modules
-  - npm install
+  - yarn --version
+  - yarn install
+  - cd embark-ui && yarn install && cd ..
+  - git status && node -e "process.exit(require('child_process').execSync('git status --porcelain').toString().trim()===''?0:1)"
```

Comment:
If after `yarn install` has been run there are unstaged changes or untracked files, etc., CI will stop and report failure. The idea is to catch situations where a commit should have been made for `yarn.lock` but wasn't.

I couldn't figure out a an equivalent to Bash's `test -z` for PowerShell or cmd.exe, would love to know if there is such a thing, but this inline node.js script does the job for now.